### DISCUSSION TITLE
实时便笺新旧接口都上WAF了

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#714] [General]
如题，实时便笺的两个接口都上了WAF，甚至把我本地IP也盾了……试了下bot好像暂时没影响，不知道是不是只给便笺加了盾

![image](https://user-images.githubusercontent.com/48074861/158173969-7d6df2d3-2ecf-4238-b656-eb7ad4b17cc9.png)

**Comment by mark9804** [comment]
我去，有点凶残

**Comment by Xm798** [comment]
额……我米游社也被盾了，离大谱了

<img src="https://user-images.githubusercontent.com/48074861/158174654-e86e38dd-2af6-4ee8-be85-ff83ed3d4e93.jpg" height="640" />
